architecture/2025-04-compute-platform-extensibility.md

```diff
@@ -0,0 +1,505 @@
+# ACI Integration
+
+* **Author**: Brooke Hamilton
+
+## Overview
+
+The Radius and ACI integration aims to enable deployments of applications and containers using the Radius control plane in Kubernetes to target ACI environments. The primary goals include setting up Radius environments targeting ACI, deploying to ACI, and managing ACI-specific configurations through punch-through configurations, including confidential containers.
+
+This document describes the initial integration with ACI, which is built upon an earlier demo capability. Our longer-term approach will provide an extensibility capability to Radius for deploying to other compute platforms, which will be described in a separate document.
+
+## Terms and definitions
+
+* ACI: Azure Container Instances, a service that allows users to run Docker containers on Azure without managing servers.
+
+## Objectives
+
+* To integrate Azure Container Instances (ACI) with Radius, enabling users to deploy and manage their applications and containers in ACI environments.
```

Comment:
What is the difference between an application and a container? An application will run in a container, right? Are these terms different in ACI?

Reply:
Yes ACI hosts containers only. I will remove the application phrase to make it more clear.